src/modules/addTask.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable import/no-cycle */
+import saveStorage from './saveStorage.js';
+import renderTasks from '../index.js';
+
+const addTask = (todoInput) => {
+  let list = JSON.parse(localStorage.getItem('todo-list'));
+
+  if (todoInput.value === '') {
+    todoInput.focus();
+  } else if (list != null) {
+    list.push({
+      description: todoInput.value,
+      completed: false,
+      index: list.length,
```

Comment:
- [ ] the property index should be set to the value of the new array length. From the screenshot below the last item in the todolist should be index two instead of 1
![image](https://user-images.githubusercontent.com/68167810/129879843-4b974f68-34df-4356-82fd-1f34c2cd5cca.png)